torch/_inductor/cudagraph_trees.py

```diff
@@ -1396,7 +1409,10 @@ def _allocate_and_copy_recording_inputs(self, inputs):
             stream=self.stream,
         ):
             for i, inp in enumerate(inputs):
-                if i not in self.static_input_idxs:
+                if not isinstance(inp, torch.Tensor):
+                    assert isinstance(inp, (int, torch.SymInt))
```

Comment:
if we see an int here, that value won't be specialized right ? don't we want only symints ?

Reply:
I suspect that the int case is impossible, but yeah, if an int does show up, that means we already specialized it and there's nothing to do